packages/crypto/packages/core/README.md

```diff
@@ -1,28 +1,37 @@
-# @iroha2/crypto-core
+# `@iroha2/crypto-core`
 
-Core Iroha v2 JS Crypto package. Contains unified crypto interface. Compiled implementations for each target are in their own packages, accordingly:
+The `@iroha2/crypto-core` package contains the unified crypto interface for Iroha 2 Javascript.
+Compiled implementations for each target (`node`, `web`, `bundler`) are in their own packages:
 
--   `@iroha2/crypto-target-node`
--   `@iroha2/crypto-target-web`
--   `@iroha2/crypto-target-bundler`
+- [`@iroha2/crypto-target-node`](../target-node/)
+- [`@iroha2/crypto-target-web`](../target-web/)
+- [`@iroha2/crypto-target-bundler`](../target-bundler/)
```

Comment:
Not sure if these links will work when this package is published. AFAIC the package's README will be wrapped with the package as-is. Then it could be displayed by the packages registry, and it will have broken links in it.

Anyway it is not a big problem now - we don't use public NPM Registry, and our Nexus Registry doesn't show READMEs. But if to fix it, I'd change it to a full GitHub link, e.g. `https://github.com/hyperledger/iroha-javascript/tree/iroha2/packages/crypto/packages/target-node`

Reply:
```suggestion
- [`@iroha2/crypto-target-node`](https://github.com/hyperledger/iroha-javascript/tree/iroha2/packages/crypto/packages/target-node)
- [`@iroha2/crypto-target-web`](https://github.com/hyperledger/iroha-javascript/tree/iroha2/packages/crypto/packages/target-web)
- [`@iroha2/crypto-target-bundler`](https://github.com/hyperledger/iroha-javascript/tree/iroha2/packages/crypto/packages/target-bundler)
```